client/packages/android/app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <resources>
-    <string name="app_name">openmsupply-client</string>
-    <string name="title_activity_main">openmsupply-client</string>
+    <string name="app_name">@openmsupply-client/android</string>
+    <string name="title_activity_main">@openmsupply-client/android</string>
```

Comment:
Is this the new format for specifying app name and title on Android ?

Reply:
it just came through when I recreated the android app. I'll revert the changes.. thanks, I missed those ones